app/src/main/java/net/gsantner/markor/format/todotxt/TodoTxtTextActions.java

```diff
@@ -68,7 +68,7 @@ protected ActionCallback getActionCallback(@StringRes int keyId) {
     @Override
     public List<ActionItem> getActiveActionList() {
 
-        final int projectIcon = R.drawable.ic_add_white_24dp;
+        final int projectIcon = R.drawable.ic_local_offer_black_24dp;
```

Comment:
How about 
![image](https://user-images.githubusercontent.com/1940940/127396064-fb00460d-7214-46e1-aa46-f0c443ca2cf9.png)

Or
![image](https://user-images.githubusercontent.com/1940940/127396213-55da2238-6679-4027-b760-6f349fe4a15f.png)

Reply:
Did you use conext/project or the default category/tag naming? I've always used the tag one, so used to this one at last

Reply:
I have always used project / context. Which is what it is in the todo.txt spec https://github.com/todotxt/todo.txt

Reply:
Though either way, since the prefix is +, I think the icon should have a + in it

Reply:
![grafik](https://user-images.githubusercontent.com/6735650/127397460-691df661-25e9-4632-a5d0-5645c2ae08cf.png)

?

Reply:
Perfect!

Reply:
shall I add? or will you?

Reply:
you can

Reply:
Actually, could you add it? I don't see this icon under asset studio

Reply:
https://fonts.google.com/icons?selected=Material+Icons

select icon, android tab on the sidebar. xml in zip